Update Soroban Getting Started (#2)
AhaLabs/soroban-docs: docs/getting-started/storing-data.mdx

### Contract Data Keys

Contract data is associated with a key. The key can be used at

> a later time to lookup the value.

Reviewer: While we're here we may as well address the "'lookup' is a noun; 'look up' is a verb" issue

```suggestion
a later time to look up the value.
```

(similar to "login" vs "log in" or "setup" vs "set up")

In this section, we added a new contract to this project, reorganizing the project as a multi-contract project using Cargo Workspaces. The new contract made use of Soroban's storage capabilities to store and retrieve data. We also learned about the different kinds of storage and how to manage their lifetimes.

> Next we'll learn a bit more about deploying contracts to Soroban's Testnet network and interact with our incrementor contractC using the CLI.

Reviewer 2: ```suggestion
Next we'll learn how to deploy our incrementor contract to Soroban's Testnet network and interact with it using the CLI.
```

Reviewer: I like the "a bit more about deploying" version better, since there was a previous step about deploying the one-step way, and the next page is about the two-step way.

Looks like there is a typo to be fixed, though. `contractC`
